contracts/interchain-token-service/src/contract/migrations/v1_1_0.rs

```diff
@@ -76,17 +76,14 @@ pub fn migrate(deps: DepsMut, msg: MigrateMsg) -> Result<(), ContractError> {
     Ok(())
 }
 
-fn convert_max_uint_to_max_bits(max_uint: nonempty::Uint256) -> Result<NumBits, ContractError> {
-    let fixed_cases = [256, 128, 64, 32];
-    for case in fixed_cases {
-        // some chains were incorrectly registered with the max number of bits as the max uint
-        if *max_uint == Uint256::from_u128(case as u128) {
-            return NumBits::try_from(case).map_err(|err| err.into());
-        }
+fn convert_max_uint_to_max_bits(max_uint: Uint256) -> NumBits {
+    if max_uint <= Uint256::from_u128(256) {
```

Comment:
Don't we allow overwriting the max unit now? Can that be used instead of having a devnet specific migration? It's low risk in production so I'm still fine with having it